### PR TITLE
[TS] LPS-156552

### PR DESCRIPTION
**Pull request by jonathanmccann**
https://issues.liferay.com/browse/LPS-156552

Please let me know if I'm misunderstanding the role of the "Client Credentials User" or if there is a better way to resolve this issue.

### CODE REVIEW

**Comment by JoyceWang08211** [comment]
The faliure of [OAuth2#ClientCredentialsShouldBeConfigurableInUserScope](https://testray.liferay.com/home/-/testray/case_results/1884749211) is caused by the error message changed when denied access permission.
The expected error message should contain **_must have DELETE permission for com.liferay.journal.model.JournalArticle_**
The current error message is `{"exception":"Access denied to com.liferay.journal.service.JournalArticleService#deleteArticle","throwable":"java.lang.SecurityException: Access denied to com.liferay.journal.service.JournalArticleService#deleteArticle","error":{"message":"Access denied to com.liferay.journal.service.JournalArticleService#deleteArticle","type":"java.lang.SecurityException"}}`

In summary, portal did deny the access permission but error message changed. If this is an expected behavior, then we need to update the related testcase.

cc @gaborlovasdotcom
